### PR TITLE
the crypto timeout is based on the last crypto packet sent

### PR DESCRIPTION
**Pull request by marten-seemann**
From reading the text, it was not clear that the timer is set based on the last (and not the first) crypto packet sent.

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -412,7 +412,7 @@ is available, or if the network changes, the initial RTT SHOULD be set to 100ms.
 When an acknowledgement is received, a new RTT is computed and the timer
 SHOULD be set for twice the newly computed smoothed RTT.
 
-When crypto packets are sent, the sender MUST set a timer for the crypto
+Every time a crypto packets is sent, the sender MUST set a timer for the crypto
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md]: I'm not sure this is significantly clearer, but I do understand what you're trying to address.

```suggestion
Every time a crypto packet is sent, the sender MUST set a timer for the crypto
```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -412,7 +412,7 @@ is available, or if the network changes, the initial RTT SHOULD be set to 100ms.
 When an acknowledgement is received, a new RTT is computed and the timer
 SHOULD be set for twice the newly computed smoothed RTT.
 
-When crypto packets are sent, the sender MUST set a timer for the crypto
+Every time a crypto packet is sent, the sender MUST set a timer for the crypto
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md]: How about "A timer MUST be updated to expire after the crypto timeout period after a new crypto packet is sent"?

Comment (mikkelfj) [reply draft-ietf-quic-recovery.md]: > If no previous RTT is available,
-->
If no previous RTT, or a-priori estimate, is available,

Comment (ianswett) [reply draft-ietf-quic-recovery.md]: @mikkelfj is this suggestion about a different paragraph?

Comment (mikkelfj) [reply draft-ietf-quic-recovery.md]: yes, I should have been more clear. It is existing text in the preceding paragraph, or thereabout.

**Pull request merged**